pkg/indexes/indexes.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Isn't the plural of index indices? :nerd_face:

Reply:
Yes, but ... we say indexes a ton when programming 😄